content/docs/actors/pay_per_result_actors.md

```diff
@@ -0,0 +1,56 @@
+
+![Free and paid actors in Apify Store]({{@asset actors/images/free_vs_paid_actors.webp}})
+
+This platform usage includes components such as [compute](https://docs.apify.com/platform/actors/running/compute-units), operations on [storages](https://docs.apify.com/platform/storage), and usage of [residential proxies](https://docs.apify.com/platform/proxy/residential-proxy) or [SERPs](<https://docs.apify.com/platform/proxy/google-serp-proxy>). You can see the usage your actor run generated on the run detail page.
+
+![Example run usage]({{@asset actors/images/example_run_usage.webp}})
+
+In this model, it is quite difficult to know beforehand exactly how much usage the actor will generate when you run it so you cannot easily estimate your costs upfront.
+
+## Paying only for results instead
+
+With **pay per result** model you will only pay for the results that an actor returns when you run it, and you will not be charged for the underlying platform usage. Technically, **results** you are charged for are the clean items in the default [dataset](https://docs.apify.com/platform/storage/dataset) created by the actor run.
+
+<!-- TODO - add a bit about the ability to set the maximum limits actor should return -->
+
+### How do I know an actor is paid per result?
+Actor that is paid per result will display the price per 1,000 items in the store.
+
+![Pay per result actor in store]({{@asset actors/images/pay_per_result_actor_store_card.webp}})
+
+When you try the actor on the platform, you will see that the actor is paid per result next to the actor name.
+
+![Actor paid per result in Console]({{@asset actors/images/console_pay_per_result_tag.webp}})
+
+### Do I need to pay monthly rental fee to run the actor?
+No, the actor is free to run, you only pay for the results.
+
+### What happens when I interact with the dataset after the run finishes?
+Under the **pay per result** model, all platform costs generated **during the run of an actor** are not charged towards your account; you pay for the results instead. After the run finishes, any interactions with the default dataset storing the results, such as reading the results or writing additional data, will incur the standard platform usage costs. But do not worry, in the vast majority of cases, you only want to read the result from the dataset and that costs near to nothing.
+
+### Do I pay for the storage of results on the Apify platform?
+You will still be charged for the timed storage of the data in the same fashion as before. You can always decide to delete the dataset to reduce your costs after you exported the data from the platform. By default, any unnamed dataset will be automatically removed after your data retention period, so it most cases this is nothing to worry about.
```

Comment:
You will still be charged for the timed storage of the data in the same fashion as before. You can always decide to delete the dataset to reduce your costs after you export the data from the platform. By default, any unnamed dataset will be automatically removed after your data retention period, so in most cases, this is nothing to worry about.